services/movie_session.py

```diff
@@ -38,7 +38,15 @@ def update_movie_session(
     if cinema_hall_id:
         movie_session.cinema_hall_id = cinema_hall_id
     movie_session.save()
+    return movie_session
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `delete_movie_session_by_id` function does not handle the case where a movie session with the specified ID does not exist. Consider using a try-except block to catch `MovieSession.DoesNotExist` exceptions and handle them appropriately.